selection.cpp

```diff
@@ -0,0 +1,28 @@
+#include<bits/stdc++.h>
+int main (){
+	int stack[1000], op, top=9999, po, aux;
+	printf("Enter the length of vector\nThe system suports until 1000 elements!\n");
+	scanf("%d", &op);
+	printf("Enter the elements!\n");
+	
+	for ( int i=0; i<op; i++){
+		scanf("%d", &stack[i]);
+	}
+	
+	for ( int i=0; i<op; i++){
+		top=stack[i];
+		po=i;
+		for ( int j=i; j<op; j++){
+			if(stack[j]<top){
+				top=stack[j];
+				po=j;		
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpgo77r2ps/selection.cpp
+++ b/tmp/tmpgo77r2ps/selection.cpp
@@ -15,7 +15,7 @@
 		for ( int j=i; j<op; j++){
 			if(stack[j]<top){
 				top=stack[j];
-				po=j;		
+				po=j;
 			}
 		}
 		aux=stack[i];
```